CalculatorEngine.java

```diff
@@ -0,0 +1,78 @@
+import java.util.Scanner;
+
+public class CalculatorEngine {
+    private int firstArabicNumber;
+    private int secondArabicNumber;
+    private Scanner input = new Scanner(System.in);
+    private Calculations calculations = new Calculations();
+    private RomanToArabicNumbersChanger romanToArabicNumbersChanger = new RomanToArabicNumbersChanger();
+    private ArabicToRomanNumbersChanger arabicToRomanNumbersChanger = new ArabicToRomanNumbersChanger();
+
+    public void displayMainMenu() throws RomanException {
+        int option;
+        System.out.println("What do you want to do?\n1. Make calculations on arabic numbers\n2. Make calculations on roman numbers.\n");
+        switch (input.nextInt()) {
+            case 1: {
+                displayCalculationsMenu();
+                option = input.nextInt();
+                getArabicNumbersToCalculate();
+                System.out.println(getResultOfCalculations(option));
```

Comment:
Do not use System.out -> Get familiar with Loggers topic (like slf4j- it also is very simple with Lombok usage)